File: 1º Semestre/Programação de Computadores/Lista de Exercicios 2/14.cpp
#include <string>
#include <iostream>
using namespace std;

int main(){
	float nota1, nota2, nota3;
	string nome;
	bool sexo;
	
	printf("Digite seu nome: ");
	scanf("%s", &nome);
	
	printf("Qual seu sexo (0: Masculino ou 1: Feminino): ");
	scanf("%b", &sexo);
	
	printf("Qual sua primeira nota? ");
	scanf("%f", &nota1);
	
	printf("Qual sua segunda nota? ");
	scanf("%f", &nota2);
	
	printf("Qual sua terceira nota? ");
	scanf("%f", &nota3);
	
	if(sexo){
		if(nota1 + nota2 + nota3 >= 6){
			printf("A aluna %s foi aprovada com media %0.2f", &nome, nota1 + nota2 + nota3);
		} else {
			printf("A aluna %s foi reprovada com media %0.2f", &nome, nota1 + nota2 + nota3);
		}
	} else {
		if(nota1 + nota2 + nota3 >= 6){
			printf("O aluno %s foi aprovado com media %0.2f", &nome, nota1 + nota2 + nota3);
		} else {
			printf("O aluno %s foi reprovada com media %0.2f", &nome, nota1 + nota2 + nota3);
		}
	}
	
}

File: 1º Semestre/Programação de Computadores/Lista de Exercicios 2/Python/14.py
"""
14. Escreva um programa em linguagem C que solicite ao usuário a média para aprovação
em um curso e em seguida solicite ao usuário o nome, sexo e as 03 notas do aluno e ao
final imprima a frase: "O aluno XXXXX foi aprovado com media YY" considerando o gênero
do(a) aluno(a) e se foi aprovado(a) ou reprovado(a)
"""

mediaInput = float(input("Qual a media para aprovação? "))
nomeInput = str(input("Qual o seu nome? "))
sexoInput = str(input("Qual o seu sexo? (masculino ou femino) ")).lower()
nota1 = float(input("Digite a primeira nota: "))
nota2 = float(input("Digite a segunda nota: "))
nota3 = float(input("Digite a terceira nota: "))
media = (nota1 + nota2 + nota3) / 3

if media >= mediaInput:
    if sexoInput != "masculino":
        print("A aluna {} foi aprovada com media {}".format(nomeInput, media))
    else:
        print("O aluno {} foi aprovado com media {}".format(nomeInput, media))
else:
    if sexoInput != "mascuino":
        print("A aluna {} foi reprovada com media {}".format(nomeInput, media))
    else:
        print("O aluno {} foi reprovado com media {}".format(nomeInput, media))